transformer_lens/ActivationCache.py

```diff
@@ -43,6 +43,39 @@ def __init__(
         self.has_embed = "hook_embed" in self.cache_dict
         self.has_pos_embed = "hook_pos_embed" in self.cache_dict
 
+    @classmethod
+    def from_caches(cls, cache_list: List[ActivationCache], in_place: Bool = True) -> ActivationCache:
+        # Constructor that combines a list of caches into a big one
+        
+        assert len(cache_list) > 0, "Cannot stack an empty list of caches"
+        assert all(
+            cache.has_batch_dim for cache in cache_list
+        ), "Cannot stack caches without batch dimensions"
+
+        # Concat a dict of lists once rather than concatting many times
+        cache_dict = {k:[v] for k,v in cache_list[0].cache_dict.items()}
+
+        if not in_place:
+            cache_dict = deepcopy(cache_dict)
+        model = cache_list[0].model.name
```

Comment:
Rename to `model_name`

Reply:
Also this doesn't work. You maybe want `model.cfg.model_name`?

Reply:
Not a big deal either way

Reply:
Thanks for noticing that!